### ISSUE TITLE
TypeError: 'class Meta' got invalid attribute(s): index_together with Django 5.1

### ISSUE THREAD

**Issue by hantsaniala** [Tivix/django-cron#250]
Temporary workaround, stick with Django 5.0.

Django-Cron version: 0.6.0 
Env manager: Pipenv
OS: Ubuntu 22.04

**Comment by jorenham** [comment]
This project appears to be abandoned. Feel free to use [my fork](https://github.com/jorenham/django-cron), which includes a fix for this.

**Comment by alexeydg** [comment]
> This project appears to be abandoned. Feel free to use [my fork](https://github.com/jorenham/django-cron), which includes a fix for this.

how do install your fork if in project was installed Tivix/django-cron ? i have error TypeError: 'class Meta' got invalid attribute(s): index_together when install your fork.

**Comment by jorenham** [comment]
```
pip install git+https://github.com/jorenham/django-cron.git
```

**Comment by alexeydg** [comment]
> ```
> pip install git+https://github.com/jorenham/django-cron.git
> ```

yes, i did it and got error class Meta' got invalid attribute(s): index_together

**Comment by zeeblaze** [comment]
> ```
> pip install git+https://github.com/jorenham/django-cron.git
> ```

Hi @jorenham I installed your for and I have been getting this error even after deleting my sqlite and all migrations
```
Django.db.utils.OperationError: index_django_cron_code_89ad04_idx already exists
```

**Comment by jorenham** [comment]
Hmm strange, I thought that https://github.com/jorenham/django-cron/pull/2 would've fixed it 🤔 . Coiuld you share the full stacktrace?

**Comment by zeeblaze** [comment]
> Hmm strange, I thought that [jorenham#2](https://github.com/jorenham/django-cron/pull/2) would've fixed it 🤔 . Coiuld you share the full stacktrace?

Ohhh, I didn't check the PR comments, I just installed directly...I will try and see if it fixes it

**Comment by pawisoon** [comment]
> ```
> pip install git+https://github.com/jorenham/django-cron.git
> ```

Can you publish it as pip package?

**Comment by jorenham** [comment]
> > ```
> > pip install git+https://github.com/jorenham/django-cron.git
> > ```
> 
> Can you publish it as pip package?

I don't see any benefit in doing that, and I have enough packages I maintain already 🤷🏻
